packages/core/src/llm/model/router.integration.test.ts

```diff
@@ -182,34 +182,30 @@ describe('ModelRouter Integration Tests', () => {
       expect(hasPirateWord).toBe(true);
     });
 
-    it.skipIf(skipInCI)(
-      'should support streaming',
-      async () => {
-        if (!process.env[envVar]) {
-          throw new Error(`${envVar} not set - required for ${provider} integration tests`);
-        }
-
-        const agent = new Agent({
-          id: 'test-agent',
-          name: 'test-agent',
-          instructions: 'You are a helpful assistant.',
-          model: modelId,
-        });
-
-        const { textStream } = await agent.stream('Count from 1 to 3');
-
-        const chunks: string[] = [];
-        for await (const chunk of textStream) {
-          chunks.push(chunk);
-        }
-
-        expect(chunks.length).toBeGreaterThan(0);
-        const fullText = chunks.join('');
-        expect(fullText).toBeDefined();
-        expect(typeof fullText).toBe('string');
-      },
-      { timeout: 30000 },
-    );
+    it.skipIf(skipInCI)('should support streaming', { timeout: 30000 }, async () => {
```

Comment:
we upgraded to vitest v4 but missed this, change with how this fn receives args, it was throwing an error